### DISCUSSION TITLE
Batch size?

### DISCUSSION THREAD

**Discussion by TKassis** [Raschka-research-group/coral-pytorch#34] [Q&A]
Thank you for this useful package. Looking at the CORN paper, and just some quick tests, do we know why CORN seems to perform slightly better with small batch sizes? I would think larger batch size would perform better especially if dealing with 50+ ordinal classes.

Do we have some intuition with regards to how number of ordinal classes might affect optimal batch size?

**Comment by rasbt** [comment]
Hi there,

that's a good question. I treated the batch size as a hyperparameter and off the top of my head, I don't have a good explanation here.

**Comment by TKassis** [comment]
Got it, thanks!
